codegen/facelift/templates/ServiceIPC.template.h

```diff
@@ -277,12 +290,15 @@ class {{interface}}IPCProxy : public facelift::IPCProxy<{{interface}}PropertyAda
         {% for property in interface.properties %}
 
         {% if property.type.is_interface %}
-        // qFatal("Property of interface type not supported");
-        qWarning() << "TODO: handle interface properties";
+        QString {{property.name}}_objectPath;
+        msg >> {{property.name}}_objectPath;
+        m_{{property.name}}Proxy.update(this, {{property.name}}_objectPath);
+        m_{{property.name}} = m_{{property.name}}Proxy.getValue();
         {% elif property.type.is_model %}
         int {{property.name}}Size;
         msg >> {{property.name}}Size;
         m_{{property.name}}.setSize({{property.name}}Size);
+
```

Comment:
Then you'll probably need an empty line in the first clause, as well (should you have several interface props).